notary/README-short.txt

```diff
@@ -0,0 +1,3 @@
+The Notary server is a front line metadata server for the Notary client.
+Notary signer is an internal service that supports Notary server by
+providing a more secure key management and signing service.
```

Comment:
Oh, damn. I thought I remembered it being 100 words (and figured it was guideline). Will come up with something shorter but it's basically going to be useless at that length.